prac_07/myguitars.py

```diff
@@ -0,0 +1,50 @@
+class Guitar:
+    """Represent a guitar with a name, year, and cost."""
+
+    def __init__(self, name, year, cost):
+        """Initialize a Guitar with a name, year, and cost."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __repr__(self):
+        """Return a string representation of the Guitar."""
+        return f"{self.name} ({self.year}), worth ${self.cost:.2f}"
+
+    def __lt__(self, other):
+        """Less-than comparison to sort Guitars by year."""
+        return self.year < other.year
+
+
+def main():
+    """Main program to read, display, and sort guitars."""
+    guitars = read_guitars()
+    print("Unsorted Guitars:")
+    display_guitars(guitars)
+
+    # Sort guitars by year (using __lt__)
+    guitars.sort()
+    print("\nGuitars Sorted by Year:")
+    display_guitars(guitars)
+
+
+def read_guitars():
+    """Read guitar data and return a list of Guitar objects."""

```

Comment:
This function was supposed to read from a csv file and create a list of guitars.